src/api/app/helpers/webui/project_helper.rb

```diff
@@ -122,4 +122,12 @@ def escape_list(list)
         "']".html_safe
     end.join(',').html_safe
   end
+
+  def show_package_actions?
+    return false if @is_maintenance_project
+    return false if @project.defines_remote_instance?
+    return true unless @is_incident_project && @packages.present? &&
+                       @has_patchinfo && @open_release_requests.empty?
+    return false
```

Comment:
@dmarcoux @Ana06 i added the ```return false```, hope its fine for both of you :smiley:

Reply:
`return` is not needed when it's the last line in Ruby as it is `implicit`

Reply:
@dmarcoux changed it